src/wizard/obtain_source.jl

```diff
@@ -258,7 +258,7 @@ function obtain_binary_deps(state::WizardState)
                     String(HTTP.get(canon_url).body)))
             elseif bindep_select == 2
                 println(state.outs, "Please provide the build.jl file. Press ^D when you're done")
-                script = String(read(STDIN))
+                script = String(read(stdin))
```

Comment:
I think this should be `state.ins`.  Feel free to change it.

Reply:
done